opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -1629,7 +1629,7 @@ const updateAttributeRaw = async (context, user, instance, inputs, opts = {}) =>
           aliasesInput.value = R.uniqBy((e) => normalizeName(e), [...aliasesInput.value, ...(instance[aliasField] || [])]);
         }
         // Internal ids alias must be generated again
-        const aliasesId = generateAliasesId(R.uniq([instance.name, ...aliasesInput.value]), instance);
+        const aliasesId = generateAliasesId(aliasesInput.value, instance);
```

Comment:
why remove instance.name ?

Reply:
As alias are now normal id, it will be a duplicate between standard_id and the alias_id generated